src/Shared/Resources/Strings.shared.resx

```diff
@@ -278,10 +278,15 @@
     <value>MSB5028: Solution filter file at "{0}" includes project "{1}" that is not in the solution file at "{2}".</value>
     <comment>{StrBegin="MSB5028: "}UE: The project filename is provided separately to loggers.</comment>
   </data>
+  <data name="WildcardResultsInDriveEnumeration" xml:space="preserve">
+    <value>MSB5029: The value "{0}" of the "{1}" attribute in element &lt;{2}&gt; is an invalid wildcard that results in drive enumeration, or enumerating all files on the drive, which was likely not intended.</value>
```

Comment:
I want to make sure this is super crisp. @chethusk, you may also have thoughts. The situation here is _usually_  something like `<Foo Include="$(NotActuallyAlwaysDefined)\**" />` (but of course they could just type `Include="C:\**"` or whatever so we can't unequivocally say that). 

```suggestion
    <value>MSB5029: The value "{0}" of the "{1}" attribute in element &lt;{2}&gt; is an wildcard that results in enumerating all files on the drive, which was likely not intended. Check that referenced properties are always defined.</value>
```